src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    try {
+      return one.mod(two).toString();
+    } catch (err) {
+      return "Can't find modulo as can't divide by 0.";
+    }
+  }
+  throw Error(`Unknown operation '${operation}'`);
```

Comment:
- [ ] I noticed that the implementation for the modulus operation using the `%` button seems to be incorrect. The code shows that the modulus operation is being performed using the `mod()` function on the one variable with the two variable as its argument. However, there is an issue with this implementation.

_The code includes a try-catch block to handle any potential errors that might occur during the calculation. This is a good practice to handle exceptions, but it seems that the specific error being caught is related to dividing by zero. The catch block returns the message "Can't find modulo as can't divide by 0." This message suggests that the code assumes that the mod() function handles division by zero gracefully. However, the modulus operation (%) should not involve division by zero in the first place._ Kindly fix this since your modulo button is not working :pray: